website/docs/r/nginx_deployment.html.markdown

```diff
@@ -139,6 +151,18 @@ A `network_interface` block supports the following:
 
 * `subnet_id` - (Required) Specify The SubNet Resource ID to this Nginx Deployment.
 
+---
+
+A `scaling` block supports the following:
+
+* `capacity` - (Required) Specify the number of NGINX capacity units for this Nginx Deployment.
```

Comment:
is there a way to add NCU docs here? I think it might help users in case they are not aware of what an NCU is?